### PR TITLE
fix python SDK PR workflow

### PR DESCRIPTION
**Pull request by ericsampson**

### CODE REVIEW

File: .github/workflows/pr-sdk-python.yml [hunk 1]
```diff
@@ -14,6 +14,5 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  run-rust-cli-tests:
+  run-python-cli-tests:
```
Comment (kbknapp) [inline .github/workflows/pr-sdk-python.yml:17]: It should be SDK tests I believe, there are no python CLI tests.

**Review by kbknapp — COMMENTED** [review]
Good catch! But I think the workflow is named incorrectly.

**Comment by ericsampson** [comment]
@kbknapp I found a couple other places that were mislabeled, take a peek now.

**Review by kbknapp — APPROVED** [review]
🚀

**Pull request merged**